control/controlclient/auto.go

```diff
@@ -139,6 +139,7 @@ type Auto struct {
 	loginGoal      *LoginGoal  // non-nil if some login activity is desired
 	inMapPoll      bool        // true once we get the first MapResponse in a stream; false when HTTP response ends
 	state          State       // TODO(bradfitz): delete this, make it computed by method from other state
+	isSleeping     bool        // whether we are ZZZing
```

Comment:
ZZZing is cute but people will have questions about what this actually means. Might be worth a longer comment.